clang/runtime/dpct-rt/include/math.hpp

```diff
@@ -667,6 +667,13 @@ inline int64_t zero_or_signed_extent(T val, unsigned bit) {
   }
   return val;
 }
+template <typename T>
```

Comment:
Can we reuse `zero_or_signed_extent`? Seems they have same functionality

Reply:
fixed